Makefile

```diff
@@ -190,6 +190,12 @@ docker-push:
 buildah-build:
 	$(SUDO_CMD) buildah bud --tag ${IMG} .
 
+# Build the code locally and build+push an image with the local binaries rather than doing a full in-container compile.
+.PHONY: buildah-dev-push
+buildah-dev-push: build
+	$(SUDO_CMD) buildah bud -f Dockerfile.dev --tag ${IMG} .
+	$(SUDO_CMD) buildah push ${IMG}
+
```

Comment:
IMO we should add this information (how a developer can use it) in the developing documentation.

Reply:
I switched the developing Hive docs ref to this fully, I can't really think of a reason not to unless someone didn't want to setup a Go env and instead wanted to do all their compiling in a Docker container, but that would be a pretty terrible idea.